strings.txt

```diff
@@ -2022,7 +2021,7 @@ ALARM_ALARM_REPEAT_DESC
 	NL	Hiermee kun je instellen of deze wekker meermaals moet afgaan. Als je 'Nee' instelt, wordt de wekker automatisch uitgeschakeld nadat het is afgegaan. De wekker gaat pas weer af als je deze weer inschakelt.
 	NO	Med denne innstillingen kan du velge om vekkingen skal spilles mer enn én gang. Hvis innstillingen settes til Nei, vil vekkingen automatisk slås av etter at den har blitt spilt. Denne vekkingen spilles altså ikke igjen før du slår den på.
 	PL	To ustawienie umożliwia określenie, czy alarm zostanie uruchomiony więcej niż raz. Po wybraniu opcji Nie alarm wyłączy się automatycznie po uruchomieniu. Ten alarm nie zostanie uruchomiony, aż do ponownego włączenia.
-	PT	Esta opção permite que escolha se o alarme se repita mais que uma vez. Se a opção for "Não" o alarme desligar-se-á depois de tocar. Não tornará a tocar até que o torne a ligar.
+	PT	Esta opção permite que escolha se o alarme se repita mais que uma vez. Se a opção for "Não" o alarme desligar-se-á depois de tocar. Não tornará a tocar até que o torne a ligar. 
```

Comment:
You can look at the changes on GitHub and you'll see the changed lines. 

If your editor can handle regexes, then you could probably search for `\s+$` and remove those. But please only do so for the lines you modified!